packages/web-components/fast-components/src/accordion/accordion.ts

```diff
@@ -0,0 +1,132 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import {
+    keyCodeArrowDown,
+    keyCodeArrowUp,
+    keyCodeEnd,
+    keyCodeHome,
+    wrapInBounds,
+} from "@microsoft/fast-web-utilities";
+import { StartEnd } from "../patterns/start-end";
+import { applyMixins } from "../utilities";
+import { FASTAccordionItem } from "./accordion-item";
+
+export enum AccordionExpandMode {
+    single = "single",
+    multi = "multi",
+}
+
+export class Accordion extends FASTElement {
+    @attr
+    public expandmode: AccordionExpandMode = AccordionExpandMode.multi;
+
+    @observable
+    public accordionItems: HTMLElement[];
+    public accordionItemsChanged(): void {
+        if (this.$fastController.isConnected) {
+            this.accordionIds = this.getHeaderIds();
+            this.setItems();
+        }
+    }
+
+    @observable
+    public activeIndicatorRef: HTMLElement;
```

Comment:
FYI It's not a problem if it's there, but you shouldn't need `@observable` on a basic ref.

Reply:
ugh this should not be here at all